src/components/ApiData.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import './ApiData.css';
+
+function Quotes() {
```

Comment:
- Optional.
Per the react conventions, I would strongly recommend renaming your rile to `Quotes.js`. The component which a file exports is named after the file itself as per the react conventions.